### DISCUSSION TITLE
How can you upgrade Tomcat for Ant Media Server Community?

### DISCUSSION THREAD

**Discussion by adrianlodin** [ant-media/Ant-Media-Server#5408] [Q&A]
Version V2.6.2 community has security vulnerabilities in Tomcat, such as https://github.com/advisories/GHSA-p22x-g9px-3945 and https://github.com/advisories/GHSA-hfrx-6qgj-fp6c, how do I upgrade Tomcat and fix them?
This was done for the Enterprise edition, after this request:
https://github.com/orgs/ant-media/discussions/5323
Is it possible to have this for community edition?

**Comment by yashtandon113** [comment]
@adrianlodin 
Yes, you can get the community edition 2.6.3 snapshot from [here](https://oss.sonatype.org/service/local/repositories/snapshots/content/io/antmedia/ant-media-server/2.6.3-SNAPSHOT/ant-media-server-2.6.3-20230728.122403-13-community.zip). This fix is merged into it.

To upgrade your server from older version to new version, check https://antmedia.io/docs/guides/upgrading-ant-media-server/

**Reply by adrianlodin** [reply]
Worked like a charm.
Thank you very much.
